src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/rewriter/identifier/UnquoteIdentifierRuleTest.java

```diff
@@ -61,6 +61,17 @@ public void selectSpecificFieldsUsingQuotedTableNamePrefix() {
         ).shouldBeAfterRewrite("SELECT bank.lastname FROM bank");
     }
 
+    @Test
+    public void queryWithQuotedAggrAndFunc() {
+        query("" +
+                "SELECT `b`.`lastname` AS `name`, AVG(`b`.`balance`) FROM `bank` AS `b` " +
+                "WHERE ABS(`b`.`age`) > 20 GROUP BY `b`.`gender` ORDER BY `b`.`age`"
```

Comment:
Please correct both queries by using `` `b`.`lastname` `` in `GROUP BY` and `ORDER BY`. In case this would fail if we add this check in semantic analyzer in future.

Reply:
It's done now, thanks! @dai-chen @abbashus

Reply:
I still see a problem here, looks like it not fixed. It should be `lastname` in `GROUP BY` and `ORDER B` instead of `age`, because as per SQL standard we cannot use a non-group by field in `SELECT`

Reply:
@abbashus You are right. fix now. thanks!